src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -3928,6 +3928,8 @@ public override DynamicMetaObject FallbackGetIndex(DynamicMetaObject target, Dyn
                 return GetIndexArray(target, indexes, errorSuggestion).WriteToDebugLog(this);
             }
 
+            var defaultMember = target.LimitType.GetCustomAttributes<DefaultMemberAttribute>(true).FirstOrDefault();
```

Comment:
Can we change this to also use `GetCustomAttribute<DefaultMemberAttribute>(inherit: true)`?

Reply:
Hmm actually it looks like `DefaultMemberAttribute` can be declared multiple times, and this method will throw in that case.  I believe I need to change the other instance to match this one instead.

Reply:
Oh, that sounds good. Thanks for clarifying.